Workbooks/Azure Active Directory Conditional Access/By continuous access evaluation (Preview)/SigninsByCAEStatus.workbook

```diff
@@ -0,0 +1,164 @@
+            "value": [
+              "value::all"
+            ],
+            "typeSettings": {
+              "limitSelectTo": 20,
+              "additionalResourceOptions": [
+                "value::all"
+              ],
+              "selectAllValue": "*",
+              "showDefault": false
+            },
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          },
+          {
+            "id": "2379b17b-1efe-41c3-8bc3-bb39422cebf7",
+            "version": "KqlParameterItem/1.0",
+            "name": "Users",
+            "type": 2,
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "query": "AADNonInteractiveUserSignInLogs\r\n| where AuthenticationProcessingDetails has \"IsCAEToken\"\r\n| summarize Count = count() by UserDisplayName\r\n| order by Count desc, UserDisplayName asc\r\n| project Value = UserDisplayName, Label = strcat(UserDisplayName, ' - ', Count, ' sign-ins'), Selected = false",
+            "value": [
+              "value::all"
+            ],
+            "typeSettings": {
+              "limitSelectTo": 20,
+              "additionalResourceOptions": [
+                "value::all"
+              ],
+              "selectAllValue": "*",
+              "showDefault": false
+            },
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "name": "parameters - 1"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "AADNonInteractiveUserSignInLogs\r\n| where AppDisplayName in ({Apps}) or '*' in ({Apps})\r\n| where UserDisplayName in ({Users}) or '*' in ({Users})\r\n| where AuthenticationProcessingDetails has \"IsCAEToken\"\r\n| parse AuthenticationProcessingDetails with * \"\\\"IsCAEToken\\\",\\\"value\\\":\\\"\" IsTokenCAE \"\\\"}]\" *\r\n| parse AuthenticationProcessingDetails with * \"\\\"Client IP Seen By Resource\\\",\\\"value\\\":\\\"\" IpSeenByRp \"\\\"}]\" *\r\n| project ['Request DateTime'] = CreatedDateTime, ['Is CAE Token'] = IsTokenCAE, ['User'] = UserDisplayName, ['Application'] = AppDisplayName, ['Resource'] = ResourceDisplayName, ['IP Address seen by IDP'] = IPAddress, ['IP Address seen by Resource'] = iff(IpSeenByRp == '', IPAddress, IpSeenByRp)\r\n",
+        "size": 0,
+        "showAnalytics": true,
+        "title": "Continuous Access Evaluation Token Report",

```

Comment:
why is this called differently at various places

Reply:
In future, we plan on adding more tables. The earlier title was for the workbook as a whole, this title is specific to the table of information. I have worded it differently now to highlight it better